carlotEvents.js

```diff
@@ -26,17 +24,18 @@ var carLot = (function(eventHandlers) {
             eventHandlers.clickEffect(currentCar, inventory, input);
 
 
-        });
-        input.addEventListener("keyup", function() {
-            var outPutTarget = currentCar.children[2];
-            outPutTarget.innerHTML = input.value;
+            input.addEventListener("keyup", function() {
+                var outPutTarget = currentCar.children[2];
+                outPutTarget.innerHTML = input.value;
 
+            });
         });
 
     }
 
-    // carLot.activateEvents();
+
     return eventHandlers;
+    carLot.activateEvents();
```

Comment:
This line isn't needed here or in carlotStyle.js (line 42). You are calling this function correctly in quiz.js so these are just unnecessary (plus they don't pass any arguments to match your parameters for the function).